core/src/test/java/io/questdb/griffin/CopyTest.java

```diff
@@ -155,288 +144,513 @@ public void testCopyFullHack2() throws Exception {
+                "copy x from 'test-quotes-big.csv' with header true timestamp 'ts' delimiter ',' " +
                 "format 'yyyy-MM-ddTHH:mm:ss.SSSUUUZ' partition by MONTH on error ABORT; ",
                 sqlExecutionContext
         );
 
-        ParallelCopyRunnable test = this::assertQuotesTableContent;
+        CopyRunnable test = this::assertQuotesTableContent;
 
-        testParallelCopy(stmt, test);
+        testCopy(stmt, test);
     }
 
     @Test
     public void testParallelCopyLogTableStats() throws Exception {
-        ParallelCopyRunnable stmt = () -> compiler.compile(
-                "copy x from '/src/test/resources/csv/test-quotes-big.csv' with parallel header true timestamp 'ts' delimiter ',' " +
+        CopyRunnable stmt = () -> compiler.compile(
+                "copy x from 'test-quotes-big.csv' with header true timestamp 'ts' delimiter ',' " +
                         "format 'yyyy-MM-ddTHH:mm:ss.SSSUUUZ' partition by MONTH on error ABORT; ",
                 sqlExecutionContext
         );
 
-        ParallelCopyRunnable test = () -> assertQuery("stage\tstatus\trows_handled\trows_imported\terrors\n" +
-                        "\tSTARTED\tNaN\tNaN\t0\n" +
-                        "ANALYZE_FILE_STRUCTURE\tSTARTED\tNaN\tNaN\t0\n" +
-                        "ANALYZE_FILE_STRUCTURE\tFINISHED\tNaN\tNaN\t0\n" +
-                        "BOUNDARY_CHECK\tSTARTED\tNaN\tNaN\t0\n" +
-                        "INDEXING\tSTARTED\tNaN\tNaN\t0\n" +
-                        "INDEXING\tFINISHED\tNaN\tNaN\t0\n" +
-                        "PARTITION_IMPORT\tSTARTED\tNaN\tNaN\t0\n" +
-                        "PARTITION_IMPORT\tFINISHED\tNaN\tNaN\t0\n" +
-                        "SYMBOL_TABLE_MERGE\tSTARTED\tNaN\tNaN\t0\n" +
-                        "SYMBOL_TABLE_MERGE\tFINISHED\tNaN\tNaN\t0\n" +
-                        "UPDATE_SYMBOL_KEYS\tSTARTED\tNaN\tNaN\t0\n" +
-                        "UPDATE_SYMBOL_KEYS\tFINISHED\tNaN\tNaN\t0\n" +
-                        "BUILD_SYMBOL_INDEX\tSTARTED\tNaN\tNaN\t0\n" +
-                        "BUILD_SYMBOL_INDEX\tFINISHED\tNaN\tNaN\t0\n" +
-                        "MOVE_PARTITIONS\tSTARTED\tNaN\tNaN\t0\n" +
-                        "MOVE_PARTITIONS\tFINISHED\tNaN\tNaN\t0\n" +
-                        "ATTACH_PARTITIONS\tSTARTED\tNaN\tNaN\t0\n" +
-                        "ATTACH_PARTITIONS\tFINISHED\tNaN\tNaN\t0\n" +
-                        "\tFINISHED\t1000\t1000\t0\n",
-                "select stage, status, rows_handled, rows_imported, errors from " + configuration.getSystemTableNamePrefix() + "parallel_text_import_log",
+        CopyRunnable test = () -> assertQuery("phase\tstatus\trows_handled\trows_imported\terrors\n" +
+                        "\tstarted\tNaN\tNaN\t0\n" +
+                        "analyze_file_structure\tstarted\tNaN\tNaN\t0\n" +
+                        "analyze_file_structure\tfinished\tNaN\tNaN\t0\n" +
+                        "boundary_check\tstarted\tNaN\tNaN\t0\n" +
```

Comment:
It seems that there's no row for boundary_check phase finish .

Reply:
Good spot. Fixed in 68dfb9c1f4741f159fe5f3c433298f95170cbafa